src/panels/meeting-times.js

```diff
@@ -1,45 +1,64 @@
 var React = require('react');
+var $ = require('jquery');
 var classNames = require('classnames');
+var time = require('../utils/time.js');
+var moment = require('moment')
 
-var meetingTimes = require('../data/meeting-times.json');
+var secrets = require('../secrets.js');
+var groupsURL = secrets.grootServicesURL + '/groups/sigs'
 
 var ROWS_PER_PAGE = 9;
+var REFRESH_TIMES_MS = 60 * 1000;
+var SWITCH_PAGE_MS = 10 * 1000;
 
 /**
  * Meeting times panel.
  */
 var MeetingTimesPanel = React.createClass({
     getInitialState: function() {
         return {
-            index: 0
+            index: 0,
+            sigs: []
```

Comment:
Also, committees don't usually meet. 
If people want to get involved in committees, they can learn more about them on the website and reach out to respective emails